zio-kafka-testkit/src/main/scala/zio/kafka/testkit/Kafka.scala

```diff
@@ -105,6 +78,29 @@ object Kafka {
 
   final case class Sasl(value: Kafka) extends AnyVal
 
+  /**
+   * Creates an in-memory Kafka instance with a random port and configured with the provided kafka broker properties.
+   */
+  def withBrokerProps(brokerProps: Ports => Map[String, String]): ZLayer[Any, Throwable, Kafka] =
```

Comment:
IMO, this should be private

```suggestion
  private def withBrokerProps(brokerProps: Ports => Map[String, String]): ZLayer[Any, Throwable, Kafka] =
```